updater/internal/rollback/rollback.go

```diff
@@ -104,8 +102,8 @@ func (r Rollbacker) Rollback() {
 }
 
 // copyFiles copies files from inputPath to output path, skipping tmpDir.
-func copyFiles(logger *zap.Logger, inputPath, outputPath, tmpDir string) error {
-	absTmpDir, err := filepath.Abs(tmpDir)
+func copyFiles(logger *zap.Logger, inputPath, outputPath, installDir string) error {
```

Comment:
similar question to the other file. it seems a little odd to be passing in the same variable twice when it's the only use case of the function. could we get way with condensing the arguments?

Reply:
done!